src/gcinfo/gcinfodumper.cpp

```diff
@@ -300,7 +300,7 @@ PORTABILITY_ASSERT("GcInfoDumper::ReportPointerRecord is not implemented on this
                 pReg = *(SIZE_T**)(pContext + rgRegisters[iReg].cbContextOffset);
                 if (iEncodedReg == iSPRegister)
                 {
-                    pReg = (SIZE_T*)((BYTE*)pRD->pCurrentContext + rgRegisters[iEncodedReg].cbContextOffset);
+                    pReg = (SIZE_T*)((BYTE*)pRD->pCurrentContext + rgRegisters[iReg].cbContextOffset);
```

Comment:
Missing definition of x86 in rgRegisters seems troublesome although it appears inevitable given the definition of Arm64VolatileContextPointer and T_KNONVOLATILE_CONTEXT_POINTERS which doesn't have x18. The fix itself is correct, but down the road, `iEncodedReg` is consumed again here `m_pfnRegisterStateChange` which has incorrect register value (>x18)